packages/graphql-cli-validate-magento-pwa-queries/package.json

```diff
@@ -32,5 +32,8 @@
   },
   "peerDependencies": {
     "graphql-cli": "^3"
+  },
+  "resolutions": {
+    "npm-run": "~5.0.0"
```

Comment:
A few questions:

* Why not pin to the "safe" version of `sync-exec` instead?
* I'm still seeing this in `yarn-lock`. Is that OK?
```sync-exec@^0.6.2:
  version "0.6.2"
  resolved "https://registry.yarnpkg.com/sync-exec/-/sync-exec-0.6.2.tgz#717d22cc53f0ce1def5594362f3a89a2ebb91105"
  integrity sha1-cX0izFPwzh3vVZQ2LzqJouu5EQU=
```

Reply:
Oh I'm dumb -- read the notice that there is _no_ safe version for sync-exec :P Still, if that's the case it shouldn't be in `yarn.lock` right?

Reply:
Looking into this now...